system-int-test/test/cmr/system_int_test/access_control/s3_bucket_route_test.clj

```diff
@@ -0,0 +1,197 @@
+(ns cmr.system-int-test.access-control.s3-bucket-route-test
+  (:require
+   [cheshire.core :as json]
+   [clj-http.client :as client]
+   [clojure.test :refer :all]
+   [cmr.access-control.test.util :as ac-util]
+   [cmr.common.util :as util :refer [are3]]
+   [cmr.mock-echo.client.echo-util :as echo-util]
+   [cmr.system-int-test.data2.core :as data-core]
+   [cmr.system-int-test.data2.umm-spec-collection :as data-umm-c]
+   [cmr.system-int-test.system :as system]
+   [cmr.system-int-test.utils.index-util :as index]
+   [cmr.system-int-test.utils.ingest-util :as ingest]
+   [cmr.system-int-test.utils.url-helper :as url]
+   [cmr.transmit.access-control :as access-control]
+   [cmr.transmit.config :as transmit-config]))
+
+(use-fixtures :each (ingest/reset-fixture {"provguid1" "PROV1"
+                                           "provguid2" "PROV2"
+                                           "provguid3" "PROV3"}
+                                          {:grant-all-search? false
+                                           :grant-all-ingest? false}))
+
+(deftest s3-buckets-route-test
+  (let [all-prov-update-group-id (echo-util/get-or-create-group
```

Comment:
Add comments on what groups is for what purpose and why you grant certain permissions. I find it hard to follow the permissions being granted. This is only related to searching aspect of the ACL permissions, why do we manipulating the ingest ACL permissions in the test as well?